ytmusicapi/mixins/playlists.py

```diff
@@ -112,10 +112,12 @@ def get_playlist(
         if not own_playlist:
             header = response["header"]["musicDetailHeaderRenderer"]
             playlist["privacy"] = "PUBLIC"
+            playlist["own_playlist"] = False
```

Comment:
I would prefer to call this `owned`, much simpler.

Also you can just set the attribute outside the if and reuse the local variable `own_playlist`, much simpler

Reply:
OK, changed